protractor/e2e/integration/secong-page.spec.ts

```diff
@@ -59,4 +59,24 @@ describe('tests for second page', ()=> {
         expect(await secondPage.infoSubmitButton.isDisplayed()).toBeTruthy();
         await secondPage.infoSubmitButton.click();
     });
-});
+
+
+    fit('Homework 2: check amount of articles in the Blog', async () => {
+        await secondPage.blogHeaderButton.click();
+        expect (await secondPage.getCurrentUrl()).toContain('/blog.ng-book.com/');
+        expect (await secondPage.articles.count()).toEqual(20);
+        expect (await secondPage.articleHeader.count()).toEqual(20);
+        expect (await secondPage.articleDescription.count()).toEqual(20);
+        expect (await secondPage.articleLink.count()).toEqual(20);
+        await secondPage.scrollToElement(secondPage.footer);
+        expect (await secondPage.firstPage.isPresent()).toBeTruthy();
+        expect (await secondPage.currentPage.isPresent()).toBeTruthy();
+        expect (await secondPage.secondPage.isPresent()).toBeTruthy();
+        expect (await secondPage.nextPage.isPresent()).toBeTruthy();
+        expect (await secondPage.lastPage.isPresent()).toBeTruthy();
+        await secondPage.secondPage.click();
+        expect (await secondPage.getCurrentUrl()).toContain('/page/2');
+        expect (await secondPage.articles.count()).toBeGreaterThan(0);
+    })
+
+});
```

Comment:
It's advisable to divide all this logic into 2 it blocks:
The first one for checking "Navigate to blog and check articles, pages navigation"
And the second one: "Navigate to blog and check articles, pages navigation"